crates/javy/src/serde/de.rs

```diff
@@ -385,8 +442,13 @@ impl<'a, 'de> de::SeqAccess<'de> for SeqAccess<'a, 'de> {
                 self.de.value = Null.into_value(self.seq.ctx().clone())
             }
             self.index += 1;
+            // Check cycles right before kicking the deserialiazation for the
+            // sequence elements.
```

Comment:
Can you clarify this comment a little? I'm not sure what _kicking_ is referring to in this case. Also `deserialiazation` -> `deserialization`.

Reply:
Not too much to clarify, other than the comment was half-baked and not ready. Fixed.